spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/web/servlet/ServletManagementChildContextConfiguration.java

```diff
@@ -165,13 +166,22 @@ public int getOrder() {
 	static class TomcatAccessLogCustomizer extends AccessLogCustomizer
 			implements WebServerFactoryCustomizer<TomcatServletWebServerFactory> {
 
+		private final ManagementServerProperties managementServerProperties;
+
+		TomcatAccessLogCustomizer(ManagementServerProperties managementServerProperties) {
```

Comment:
You only have changed it for Tomcat. This change also has to be applied to Undertow and Jetty.